dna/codon.go

```diff
@@ -365,6 +365,19 @@ func TranslateSeqToTer(b []Base) []AminoAcid {
 	return answer
 }
 
+//TranslateToTer takes a sequence of in-frame DNA bases and translates to the first stop codon. Returns a slice of amino acids the distance to the stop codon in base zero.
+func TranslateToTer(b []Base) []AminoAcid {
+	var answer []AminoAcid
+	codons := BasesToCodons(b)
+	for i := 0; i < len(codons); i++ {
+		answer = append(answer, TranslateCodon(codons[i]))
+		if answer[len(answer)-1] == Stop {
+			break
```

Comment:
Is it more clear to return right here rather than breaking to then end up at the later return?